.gitlab/issue_templates/Documentation.md

```diff
@@ -0,0 +1,13 @@
+## Description
```

Comment:
I'm refusing to maintain this .gitlab directory. PyFunceble - until I change my mind - will only be maintained on GitHub. 

Right now GitLab is unusable for me. I have over 500+ \@\mentions across GitHub and GitLab. GitHub gives me inbox transparency. GitLab gives me headache. 

I'm ready to provide support for GitLab CI. But I'm not ready to accept this change here.

Reply:
This is not the changes I made.... https://mypdns.org/spirillen/PyFunceble/-/commit/a2ff7278884be2c6cbda05cb3d753ef505611446

Reply:
AAhhh now I can remember this one, This one is mostly intended for my use in my doc repo. not as such for the public

> Right now GitLab is unusable for me. I have over 500+ @\mentions across GitHub and GitLab. GitHub gives me inbox transparency. GitLab gives me headache.

I a knowledge this is that GH rules on this point as GL i more bound to mails than a internal "mailbox"

You can try and see my number
![image](https://user-images.githubusercontent.com/44526987/146678984-80f308c0-eded-4290-a1f8-7030723d3b8c.png)

I'm sticking to the mailbox, to find new relevant issues.